src/containers/AccountManagement.jsx

```diff
@@ -53,10 +54,30 @@ export default class AccountManagement extends Component {
       })
   }
 
+  validInfos (input) {
+    const { t } = this.context
+    // Check if field is empty or not
+    if (input.target.value !== '') {
```

Comment:
@GoOz @gregorylegarec Both are the same thing here. We just have to code in a readable way. `if` with returns is good here for validation, but will be less readable for more complexe structure. So both could be acceptable here, but  we could do keep simpler but using the @gregorylegarec way since @GoOz want to return a boolean anyway.